### PR TITLE
Show fx57 label and boost for mozilla signed add-ons

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Refs #6424 (and as the second part fixes it)

This felt a bit too straight forward so please let me know if I missed anything.

### CODE REVIEW

**Comment by diox** [comment]
@EnTeQuAk you also need to apply the `firefox57` tag when those extensions are approved and in the task to backfill the tag to existing extensions.

**Comment by EnTeQuAk** [comment]
> @EnTeQuAk you also need to apply the firefox57 tag when those extensions are approved and in the task to backfill the tag to existing extensions.

Ha, I forgot about that. I'll add that to the PR, hold on reviewing it till then :)

File: src/olympia/addons/management/commands/process_addons.py [hunk 1]
```diff
@@ -31,8 +31,12 @@
     'sign_addons': {'method': sign_addons, 'qs': []},
     'add_firefox57_tag_to_webextensions': {
         'method': add_firefox57_tag,
-        'qs': [Q(status=amo.STATUS_PUBLIC,
-                 _current_version__files__is_webextension=True)]},
+        'qs': [
+            Q(status=amo.STATUS_PUBLIC) & (
+                Q(_current_version__files__is_webextension=True) |
+                Q(_current_version__files__is_mozilla_signed_extension=True)
```
Comment (EnTeQuAk) [inline src/olympia/addons/management/commands/process_addons.py]: We could actually filter this by `_current_version__apps__max__version_int__gte=firefox_57_star` but I wonder if that's correct, hmm.

**Comment by EnTeQuAk** [comment]
Should be ready for r? now. Still unsure if we should filter the firefox57 tag by max-version for mozilla signed add-ons but I just assume that all the add-ons we put out there that have that signature are properly valid for most current firefox-versions anyway.

**Comment by diox** [comment]
You also need to change the conditions under which the tag is added in `ReviewBase.process_public()` for future uploads

File: src/olympia/addons/management/commands/process_addons.py (hunk 1 above)
Comment (diox) [reply src/olympia/addons/management/commands/process_addons.py]: it's fine like this IMHO

**Comment by EnTeQuAk** [comment]
> You also need to change the conditions under which the tag is added in ReviewBase.process_public() for future uploads

If I understand that system correctly that should be fixed in f28f6476e2

File: src/olympia/addons/management/commands/process_addons.py (hunk 1 above)
Comment (EnTeQuAk) [reply src/olympia/addons/management/commands/process_addons.py]: mhmm, well, we should set `strict_compatibility` to whatever is set in their manifest for existing ones. I'll add a task for that on Monday, that's a very good point.

**Comment by groovecoder** [comment]
Yay! Looking forward to trying this out on Thursday. Thanks!

**Pull request merged**